### PR TITLE
Carga

### PR DESCRIPTION
**Pull request by JHAIR**
Revisen porfavor

### CODE REVIEW

**Comment by FreddyClaros** [comment]
> Revisen porfavor

I think you forgot to put people for review, I will added the team, let me know if you agree or you're against it

File: MACHINE_LEARNING_SERVICE/src/controller/apis/endpointface.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#
+
+import os
+
+class EndPointConverter:
+    def __init__(self, request, save_location):
+        self.request = request
+        self.save_location = save_location
+
+    def Upload(self):
```
Comment (FreddyClaros) [inline MACHINE_LEARNING_SERVICE/src/controller/apis/endpointface.py:21]: I thin "Upload" must be lowercase

**Review by FreddyClaros — APPROVED** [review]
Looks great, but I think the images are not necessary, still looks good to me

File: MACHINE_LEARNING_SERVICE/prueba_emotion.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# 
+# @object_result.py Copyright (c)
```
Comment (FreddyClaros) [inline MACHINE_LEARNING_SERVICE/prueba_emotion.py:2]: changed to "emotion_test"

Comment (FreddyClaros) [reply MACHINE_LEARNING_SERVICE/prueba_emotion.py:2]: Also the file must be in english

Comment (FreddyClaros) [reply MACHINE_LEARNING_SERVICE/prueba_emotion.py:2]: And it must be in test folder

**Review by FreddyClaros — COMMENTED** [review]
There some thing that need to be changed, still good job

File: MACHINE_LEARNING_SERVICE/src/controller/apis/endpointface.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# 
+# @object_result.py Copyright (c)
```
Comment (FreddyClaros) [inline MACHINE_LEARNING_SERVICE/src/controller/apis/endpointface.py:2]: put the file name here please, and the file name must be something like "endpoint_face.py"

File: MACHINE_LEARNING_SERVICE/src/model/face_emotion.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# 
+# @object_result.py Copyright (c)
```
Comment (FreddyClaros) [inline MACHINE_LEARNING_SERVICE/src/model/face_emotion.py:2]: this must be "face_emotion.py"

**Review by FreddyClaros — COMMENTED** [review]
I added some comments, please resolve them, and good job

**Comment by daylerVMT** [comment]
are you uploading the .jpg files too, are you sure? then it looks good.

**Review by abrahamUrquiola — APPROVED** [review]

**Review by Reynel7 — APPROVED** [review]

**Pull request merged**